README.md

```diff
@@ -457,6 +498,45 @@ class DeliverViaSMSC
   end
 end
 ```
+
+### Custom Retry Queues
+
+By default, when a job is retried, it is added to the `@queue` specified in the worker. However, you may want to push the job into another (lower or higher priority) queue when the job fails. You can do this by dynamically specifying the retry queue. For example:
+
+```ruby
+class ExampleJob
+  extend Resque::Plugins::Retry
+  @queue = :testing
+  @retry_delay = 1
+
+  def self.work(*args)
+    Resque.enqueue_to(
```

Comment:
Something seems wrong w/ this example method, it is referencing a number of vars that aren't declared, please unpack